sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
 
 ### Other Changes
 
+## 11.4.3 (2021-09-09)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Upgraded `azure-core` and other dependencies for the library.
```

Comment:
In previous releases, we've called out all dependencies and their versions. We should be consistent and is useful to track down when the dependency version was bumped.

Reply:
This was done with the auto-generated tool, was keeping it inline with that. I agree that they should be called out more discretely.